xclim/data/fr.json

```
@@ -912,6 +912,42 @@
     "title": "Date du fin du couvert de neige",
     "abstract": "Première date à partir de laquelle l'épaisseur de neige est sous à un seuil donné pendant un nombre de jours consécutifs suite au début du couvert de neige."
   },
+  "SNOW_DEPTH_COVER_DURATION": {
+    "long_name": "Nombre de jour où l'épaisseur de neige est au-dessus d'un seuil",
+    "description": "Nombre {freq:m} de jours où l'épaisseur de neige est au-dessus ou égale à {thresh}.",
+    "title": "Durée du couvert de neige",
```

I think it's ok that the long names are the same, but the title should be unique:
```suggestion
    "title": "Durée du couvert de neige (épaisseur)",
```

---

Title should be unique, meaning
A) Two indicators can't have the same title
B) Within one indicator, we can't have "title" = "long name"

I think you mean A)? In this case, we might have a problem. `SNOW_COVER_DURATION` will be deprecated in favour of `SNOW_DEPTH_COVER_DURATION`, but in the meantime, they should have identical descriptions. Perhaps I'm duplicating code where it's not necessary?

---

Perhaps it's all right, for the title  1) `snow_cover_duration` ; 2) `snow_depth_cover_duration`; 3) `snow_amount_cover_duration`, we can use:
1) "Durée du couvert de neige"
2) "Durée du couvert de neige (épaisseur)"
3) "Durée du couvert de neige (quantité)"

and things should be fine? I still wonder how deprecation should be handled if it's the case you can't duplicate a title.

---

I meant A indeed, but this is not a strict rule (nothing should fail because of it). So, it seems to me that `snow_cover_duration` should be "Durée du couvert de neige (épaisseur)", since that's what it is?

---

As long as there is a deprecation warning for the indicator, we should be fine?

---

All right, indicators for `snow_cover_duration` and `snd_season_length` (and so on) have the same title.